packages/suite/src/actions/suite/modalActions.ts

```diff
@@ -40,6 +40,18 @@ export const onReceiveConfirmation = (confirmation: boolean) => (dispatch: Dispa
 
     dispatch(onCancel());
 };
+export const onReceiveAccount = (accountIndex: number | null) => (dispatch: Dispatch) => {
+    if (accountIndex === null) {
+        TrezorConnect.cancel();
```

Comment:
this might cause problmes with bridge 2.0.27 but I'd say we may ignore it since it is going to be deprecated soon